devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -94,6 +94,24 @@ metadata {
 
 	main(["switch","power","energy"])
 	details(["switch", "power", "energy", "refresh", "reset"])
+
+	preferences {
+			section {
+				input(
+						title: "Settings Available For Aeotec Nano Dimmer Only",
+						type: "paragraph",
+						element: "paragraph"
+				)
+				input(
+						title: "Set the MIN brightness level (Aeotec Nano Dimmer Only):",
+						description: "Adjust the minimum dimming level. This may be needed for incandescent and non-incandescent bulbs.",
```

Comment:
Since it also happens for incandescent bulbs, it might make sense to say

"This may need to be adjusted for bulbs that are not dimming properly."